dora-metrics/pelorus
# fix: Update script for docs/index.md file

## Describe the behavior changes introduced in this PR

This PR fixes some markdown links to point to the [Project documentation](https://pelorus.readthedocs.io/en/latest/) instead of the markdown files that generate the documentation. It also introduces a documentation check for PRs, to check if the `docs/index.md` file is up to date with the `README.md` file. This is done by the `.github/workflows/check-doc-updates.yml` pipeline and the Automation Command Line Interface (CLI).

## Linked Issues

No linked issues.

## Testing Instructions

To check the CLI help, run 
```
scripts/automation.py
```
To check the update documentation command help, run
```
scripts/automation.py update -h
```
There is no need to the update documentation script to be a CLI, it can a single file like `scripts/view-grafana-queries.py`. Or if you like the idea, the CLI can integrate the previous script, for example.
